apps/site/src/app/schedule/components/EventRegular.tsx

```diff
@@ -0,0 +1,122 @@
+import dayjs from "dayjs";
+import relativeTime from "dayjs/plugin/relativeTime";
+import convertToPST from "@/lib/utils/convertToPST";
+dayjs.extend(relativeTime);
+
+const dateTimeFormat = new Intl.DateTimeFormat("en", {
+	hour: "numeric",
+	minute: "numeric",
+});
+
+interface EventProps {
+	now: Date;
+	title: string;
+	eventType: string;
+	location?: string | undefined;
+	virtual?: string | undefined;
+	startTime: Date;
+	endTime: Date;
+	organization?: string | undefined;
+	hosts?: string[] | undefined;
+	description: JSX.Element;
+}
+
+interface EventTypeProps {
+	eventType: string;
+}
+
+interface EventMomentProps {
+	now: Date;
+	startTimeInPST: Date;
+	endTimeInPST: Date;
+}
+
+function EventTypeComponent({ eventType }: EventTypeProps) {
+	if (eventType === "Main") {
+		return (
+			<div className="inline-block px-4 py-1.5 font-semibold text-[#0D272D] bg-[#DFBA73] rounded-2xl sm:py-2">
+				{eventType}
+			</div>
+		);
+	} else if (eventType === "Workshop") {
+		return (
+			<div className="inline-block px-4 py-1.5 font-semibold text-[#0D272D] bg-[#94A9BD] rounded-2xl sm:py-2">
+				{eventType}
+			</div>
+		);
+	} else if (eventType === "Social") {
+		return (
+			<div className="inline-block px-4 py-1.5 font-semibold text-[#0D272D] bg-[#DFA9A9] rounded-2xl sm:py-2">
+				{eventType}
+			</div>
+		);
+	}
+}
```

Comment:
Issue: The only difference between the returned elements is their background color. You could simplify this even further by initializing a string variable, setting it to be the appropriate color based on `eventType`, and then adding that variable to the `className` string.

Reply:
I tried doing this before which worked locally, but the Vercel preview wasn't able to set the right background color
594aca7cd2747c38a801af4a902307d38fafde39

Reply:
Just found out that it's actually not recommended to dynamically declare classes like this because Tailwind won't be able to [find them](https://tailwindcss.com/docs/content-configuration). In that case, please use inline CSS and set the background color that way.